stl/inc/iosfwd

```diff
@@ -192,6 +192,16 @@ template <class _Elem, class _Traits = char_traits<_Elem>, class _Alloc = alloca
 class basic_ostringstream;
 template <class _Elem, class _Traits = char_traits<_Elem>, class _Alloc = allocator<_Elem>>
 class basic_stringstream;
+#if _HAS_CXX23
+template <class _Elem, class _Traits = char_traits<_Elem>>
+class basic_spanbuf;
+template <class _Elem, class _Traits = char_traits<_Elem>>
+class basic_ispanstream;
+template <class _Elem, class _Traits = char_traits<_Elem>>
+class basic_ospanstream;
+template <class _Elem, class _Traits = char_traits<_Elem>>
+class basic_spanstream;
+#endif // _HAS_CXX23

```

Comment:
`<iosfwd>` is missing `spanstream`, `wspanstream`, etc.

Reply:
Good catch, I thought they should be located in `<spanstream>`

Reply:
Now that I checked we only have an TR1 test for `<iosfwd>` is that something we should add?

Reply:
Yeah, it would make sense to add test coverage to `std` for `<iosfwd>` - the only reason we didn't have any is that the header hasn't received extensive changes recently, and we add test coverage to `std` on an as-needed basis.

(If you're asking whether we should modify `tr1`, we generally refrain from adding test coverage to that legacy suite, although we'll fix it in response to breaking changes. In rare situations, adding test coverage to `tr1` makes sense as the easiest option, e.g. it had infrastructure for testing random number engines, where just adding a few lines was easier than replicating that machinery in `std`.)

Reply:
Added a test for the forward declarations